package.json

```diff
@@ -11,6 +11,7 @@
   },
   "scripts": {
     "lint": "([ \"$CI\" = true ] && eslint --quiet -f codeframe . || eslint .)",
+    "winlint": "( set CI=true  && eslint --quiet -f codeframe . || eslint .)",
```

Comment:
`set CI=true` always return true, so `eslint --quiet -f codeframe .` is always executed, and `eslint .` may be executed according to the previous execution

Reply:
Sorry，I did not understand clearly.@liudonghua123  Please take a look, is this correct?

Reply:
I think it is not correct, maybe this is right.

```
if %CI% equ true ( eslint --quiet -f codeframe . ) else ( eslint . )
```

![image](https://user-images.githubusercontent.com/2276718/48927398-7c604800-ef10-11e8-984a-1739516542ff.png)